fenix/app/.experimenter.yaml

```diff
@@ -36,6 +36,9 @@ juno-onboarding:
   hasExposure: true
   exposureDescription: ""
   variables:
+    cards:
+      type: json
+      description: "Collection of user facing onboarding 'views'."
```

Comment:
🔍 Should we call it cards, since it's called cards above? Views can be confused to the individual view, wdyt?